src/com/rcv/RawElectionConfig.java

```diff
@@ -34,6 +37,8 @@
   public List<Candidate> candidates;
   // list of all cast vote record source files
   public List<CVRSource> cvrFileSources;
+  // should we report round-by-round results by precinct also?
+  public Boolean tabulateByPrecinct;
 
```

Comment:
So.... to use native boolean or not?  I'm fine with either but would like to be consistent.

Reply:
Oops. I had assumed that we had to handle the possibility of a null for this, but it turns out that's not the case: if you omit a boolean field from the config file, the parser will just default it to false.